.github/workflows/elixir.yml

```diff
@@ -27,22 +27,31 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    
     - name: Set up Elixir
       uses: actions/setup-elixir@v1
       with:
-        elixir-version: '1.10.4' # Define the elixir version [required]
-        otp-version: '23' # Define the OTP version [required]
+        elixir-version: '1.11.2'
+        otp-version: '23'
+
     - name: Restore dependencies cache
       uses: actions/cache@v2
       with:
         path: deps
         key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
         restore-keys: ${{ runner.os }}-mix-
+
     - name: Install dependencies
       run: mix deps.get
+
+    - name: Start stripe-mock
```

Comment:
The stripe mock we were using before only had endpoints for like 3 Stripe functions. This package is a full distribution straight from Stripe.

The downside right now is that means you'll need `stripe-mock` running locally or the payment tests wont pass.